### DISCUSSION TITLE
Possible to add JIT entitlement on root access Flycast ipa?

### DISCUSSION THREAD

**Discussion by ansinerd** [flyinghead/flycast#855] [Ideas]
As everyone knows since Linus discoverer that core trust bug in certain iOS versions, Sideloading for large no of users has become easier. Trollstore uses coretrustbug to sign apps as system apps. 
UTM and Pojavalaumcher team has been able to add a method to enable JIT on trollstore versions for native JIT without having to enable manually.
Another reddit user was able to add similar entitlement in dolphin ios ipa and also works.
Recently dolphinios main developers talked about officially releasing ipa for trollstore users that will have native jit enabled.

 There are thousands of people who would love to have similar JIT functionality for flycast.
Would it be possible to have such entitlement in flycast for ios? 

I am attaching link to post on how people achieved native jit entitlement on pojava and utm

https://www.reddit.com/r/jailbreak/comments/xcd2v4/news_found_way_to_achieve_wx_jit_for/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

https://www.reddit.com/r/Trollstore/comments/ydure2/trollstorejitenabler/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

Thanking in advance. Its just a polite request or discussion if its possible.

**Comment by vkedwardli** [comment]
Yes it is possible, but I thought you can switch between the user mode and  `platform-application` mode by trollstore?
https://github.com/opa334/TrollStore/issues/75

**Reply by ansinerd** [reply]
Yes we can move between user mode and system mode but somehow flycast would still crash even on user mode when I try to launch game.
Something is beyond my understanding. Although there’s way around which pojavalaumcher dev, utm dev and now dolphinios dev are implementing specifically for trollstore build that will debug jit automatically whenever launched just like jailbroken device. As mentioned in posts above. 
as of now we neee altstore/sidestore/jitterbug to launch flycast.
